js-i2b2/cells/PM/PM_ctrlr.js

```diff
@@ -543,13 +836,14 @@ i2b2.PM._processLaunchFramework = function () {
 			delete i2b2[cellKey];
 		}
 	}
-
+/* Legacy SHRINE code - Removed 6/3/16
```

Comment:
Is this your comment or the SHRINE developers? If the SHRINE developers made this, why is it in our patch?